src/data/ability.ts

```diff
@@ -4764,6 +5007,11 @@ export function applyPostSetStatusAbAttrs(attrType: Constructor<PostSetStatusAbA
   return applyAbAttrsInternal<PostSetStatusAbAttr>(attrType, pokemon, (attr, passive) => attr.applyPostSetStatus(pokemon, sourcePokemon, passive, effect, simulated, args), args, false, simulated);
 }
 
+export function applyPostDamage(attrType: Constructor<PostDamageAbAttr>,
```

Comment:
```suggestion
export function applyPostDamageAbAttrs(attrType: Constructor<PostDamageAbAttr>,
```
Not what I meant earlier 😖 

- The apply functions within `XAbAttrs` are usually called `applyX` (e.g. `PostDefendAbAttr` -> `applyPostDefend`)
- The external functions that invoke attributes are usually called `applyXAbAttrs` (e.g. `applyPostDefendAbAttrs`)

Reply:
fixed! Thanks for the explanation on that